exchanges/bybit/bybit_websocket.go

```diff
@@ -730,9 +661,6 @@ const subTplText = `
 				{{- $p }}
 				{{- $.PairSeparator }}
 			{{- end }}
-		{{- else }}
-			{{- $name }}
-			{{- if and (isCategorisedChannel $name) ($categoryName := getCategoryName $asset) -}} . {{- $categoryName -}} {{- end }}
```

Comment:
🚧 I think this means we should be removing `isCategorisedChannel`.
I can't see it used anywhere. Sorry if I've missed it.